q2_feature_classifier/_skl.py

```diff
@@ -71,25 +109,26 @@ def _predict_chunk_with_conf(pipeline, separator, confidence, chunk):
 
     y = pipeline.classes_[prob_pos.argmax(axis=1)]
 
+    taxonomy_tree = _TaxonPredictionNode.create_tree(pipeline.classes_)
```

Comment:
As you state in the comments, you assume that `pipeline.classes_` is sorted. The sklearn classes I've looked at do sort their classes, but please add an assertion here to check that it is not violated in the future.

Reply:
Since the tree structure requires the classes to be sorted, I put the assertion inside the `create_tree` method.